langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/MistralAiChatModel.java

```diff
@@ -169,16 +172,7 @@ private Response<AiMessage> generate(List<ChatMessage> messages,
         );
     }
 
-    public static MistralAiChatModelBuilder builder() {
-        for (MistralAiChatModelBuilderFactory factory : loadFactories(MistralAiChatModelBuilderFactory.class)) {
-            return factory.get();
```

Comment:
is this removal intended?
same with all other classes below

Reply:
ohh I though ( I wrongly assumed it by mistake) that it was part of the refactoring plan  bcs the anthropic module haven't spi folder and Builder factories, if it is not, I can put back it. Do you want me to put back it? is it used by spring and quarkus extensions?

Reply:
Right, Anthropic does not have `spi` directory (yet). Please put it back as it is used by Quarkus extension